### PR TITLE
Canvas art

### PR DESCRIPTION
**Pull request by BGS89**
# Zero To Mastery - Canvas 2022 🎉🎉

👉 _**Please remove the below and replace with your own values, leaving the headers where they are.**_ 👈

## PR contains:
A happy face(Please be as descriptive as possible) 🤜

## Breaking changes
None

## Screenshots
icon

### CODE REVIEW

File: src/art/BGS89/index.html [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title></title>
```
Comment (sophiabrandt) [inline src/art/BGS89/index.html:7]: Please fill out the title.

File: src/art/BGS89/script.js [hunk 2]
```diff
@@ -0,0 +1,60 @@
+function draw() {
+    const canvas = document.getElementById('test');
+    if (canvas.getContext) {
+        const ctx = canvas.getContext("2d");
+
+        // ctx.fillStyle = 'rgb(255, 0, 0)';
```
Comment (sophiabrandt) [inline src/art/BGS89/script.js:6]: Please remove unused code.

File: src/art/BGS89/style.css [hunk 3]
```diff
@@ -0,0 +1,4 @@
+canvas {
+    border: 1px solid black;
+    /* background: rgb(0, 28, 85); */
```
Comment (sophiabrandt) [inline src/art/BGS89/style.css:3]: Same here.

**Review by sophiabrandt — CHANGES_REQUESTED** [review]
`meta.json` missing, plus see other comments.

**Comment by BGS89** [comment]
I have made the changes requested

**Pull request merged**